templates/addEvent.phtml

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>Add Event</title>
+        <link rel="stylesheet" type="text/css" href="css/genericStyles.css"/>
+        <link rel="stylesheet" href="css/addEventPage.css">
+    </head>
+    <body>
+        <main class="addEventPageWrapper">
+            <h2 id="heading">Add Event</h2>
+            <form action="/addEvent" method="POST" id="addEventForm">
```

Comment:
Form will use ajax to submit so no need for action or method on the form tag

Reply:
Solved in ae3e10f